<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<!-- saved from url=(0030)https://sarwanpasha.github.io/ -->
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
    
    <link src="images/cthulu.png" rel="shortcut icon" type="image/x-icon">

    <title>Sarwan Ali</title>
    <style type="text/css">
        * {
            margin: 0;
            padding: 0;
        }

        body {
            font: 16px Helvetica, Sans-Serif;
            line-height: 24px;
            background: url(images/noise.jpg);
        }

        .clear {
            clear: both;
        }

        #page-wrap {
            width: 800px;
            margin: 40px auto 60px;
        }

        #pic {
            float: right;
            margin: -30px 0 0 0;
        }

        h1 {
            margin: 0 0 16px 0;
            padding: 0 0 16px 0;
            font-size: 42px;
            font-weight: bold;
            letter-spacing: -2px;
            border-bottom: 1px solid #999;
        }

        h2 {
            font-size: 20px;
            margin: 0 0 6px 0;
            position: relative;
        }

            h2 span {
                position: absolute;
                bottom: 0;
                right: 0;
                font-style: italic;
                font-family: Georgia, Serif;
                font-size: 16px;
                color: #999;
                font-weight: normal;
            }

        p {
            margin: 0 0 16px 0;
        }

        a {
            color: #999;
            text-decoration: none;
            border-bottom: 1px dotted #999;
        }

            a:hover {
                border-bottom-style: solid;
                color: black;
            }

        ul {
            margin: 0 0 32px 17px;
        }

        #objective {
            width: 500px;
            float: left;
        }

            #objective p {
                font-family: Georgia, Serif;
                font-style: italic;
                color: #666;
            }

        dt {
            font-style: italic;
            font-weight: bold;
            font-size: 18px;
            text-align: right;
            padding: 0 26px 0 0;
            width: 150px;
            float: left;
            height: 100px;
            border-right: 1px solid #999;
        }

        dd {
            width: 600px;
            float: right;
        }

            dd.clear {
                float: none;
                margin: 0;
                height: 15px;
            }
    </style>
	
	<style>
	/* Add a black background color to the top navigation */
.topnav {
  background-color: #333;
  overflow: hidden;
}

/* Style the links inside the navigation bar */
.topnav a {
  float: left;
  color: #f2f2f2;
  text-align: center;
  padding: 14px 16px;
  text-decoration: none;
  font-size: 17px;
}

/* Change the color of links on hover */
.topnav a:hover {
  background-color: #ddd;
  color: black;
}

/* Add a color to the active/current link */
.topnav a.active {
  background-color: #4CAF50;
  color: white;
}
	</style>
	
	<style>
        body {
            font-family: Arial, sans-serif;
            margin: 0;
            padding: 20px;
        }
        table {
            width: 100%;
            border-collapse: collapse;
        }
        th, td {
            padding: 8px;
            text-align: left;
            border-bottom: 1px solid #ddd;
        }
        th {
            background-color: #f2f2f2;
        }
    </style>
	
    <script>
        function countdown() {
            var deadlines = document.getElementsByClassName('countdown');
            for (var i = 0; i < deadlines.length; i++) {
                var deadlineDate = new Date(deadlines[i].dataset.deadline);
                var currentDate = new Date();
                var timeRemaining = deadlineDate - currentDate;

                if (timeRemaining > 0) {
                    var days = Math.floor(timeRemaining / (1000 * 60 * 60 * 24));
                    var hours = Math.floor((timeRemaining % (1000 * 60 * 60 * 24)) / (1000 * 60 * 60));
                    var minutes = Math.floor((timeRemaining % (1000 * 60 * 60)) / (1000 * 60));
                    var seconds = Math.floor((timeRemaining % (1000 * 60)) / 1000);

                    deadlines[i].innerHTML = days + "d " + hours + "h " + minutes + "m " + seconds + "s";
                } else {
                    deadlines[i].innerHTML = "Deadline passed";
                }
            }

            setTimeout(countdown, 1000);
        }

        window.onload = countdown;
    </script>
</head>

<body data-new-gr-c-s-check-loaded="14.990.0" data-gr-ext-installed="">
    <div id="page-wrap">

       

	   <div class="topnav">
  <a href="https://sarwanpasha.github.io/">Home</a>
  <a href="https://sarwanpasha.github.io/Sarwan_Ali_CV.pdf" target="_blank">CV</a>
  <a href="https://scholar.google.com/citations?user=9dtXSoAAAAAJ&hl=en" target="_blank">Google Scholar</a>
  <!-- <a href="https://sarwanpasha.github.io/conferences.html">Conferences</a> -->
  <a href="https://sarwanpasha.github.io/Courses/course_home.html">Teaching</a>
  <a href="https://sarwanpasha.github.io/PapersDiscussion.html">Papers Discussion</a>
  
</div>
 <br>
  <br>

  <!-- <td>Here I discuss interesting papers of mine and from other authors</td> -->
  <h2>Machine Learning for Genomics</h2>
  <!-- <h1>Interesting Conferences</h1> -->
    <table>
        <thead>
            <tr>
                <th>Topic</th>
                <th>Slides</th>
                <th>Video</th>
            </tr>
        </thead>
        <tbody>
            <tr>
                <td><a style="color:violet">Topic 1.1: Introduction to Genomics and ML</a>
		<br>
		<strong>Central dogma of molecular biology review</strong>
		</td>
                <td> <a class="email" href="https://sarwanpasha.github.io/Courses/ML_For_G/IGML_1.pdf" target="_blank" style="color:blue">Slides</a>  </td>
                <td> <a> - </a> </td>
            </tr>

	    <tr>
                <td><a style="color:violet">Topic 1.2: Introduction to Genomics and ML</a>
		<br>
		<strong>Types of genomic data: DNA sequences, RNA-seq, ChIP-seq, ATAC-seq</strong>
		</td>
                <td> <a class="email" href="https://sarwanpasha.github.io/Courses/ML_For_G/IGML_2.pdf" target="_blank" style="color:blue">Slides</a>  </td>
                <td> <a> - </a> </td>
            </tr>

	    <tr>
                <td><a style="color:violet">Topic 1.3: Introduction to Genomics and ML</a>
		<br>
		<strong>Overview of machine learning in biological contexts</strong>
		</td>
                <td> <a class="email" href="https://sarwanpasha.github.io/Courses/ML_For_G/IGML_3.pdf" target="_blank" style="color:blue">Slides</a>  </td>
                <td> <a> - </a> </td>
            </tr>

	    <tr>
                <td><a style="color:violet">Topic 1.4: Introduction to Genomics and ML</a>
		<br>
		<strong>Key databases and resources (NCBI, Ensembl, UCSC Genome Browser)</strong>
		</td>
                <td> <a class="email" href="https://sarwanpasha.github.io/Courses/ML_For_G/IGML_4.pdf" target="_blank" style="color:blue">Slides</a>  </td>
                <td> <a> - </a> </td>
            </tr>	    

	    <tr>
                <td><a style="color:green">Topic 2.1: Mathematical Foundations</a>
		<br>
		<strong>Linear algebra for genomics applications</strong> 
		</td>
		<td> <a class="email" href="https://sarwanpasha.github.io/Courses/ML_For_G/MF_1.pdf" target="_blank" style="color:blue">Slides</a>  </td>
                <td> <a> - </a> </td>
           </tr>	    

	    <tr>
                <td><a style="color:green">Topic 2.2: Mathematical Foundations</a>
		<br>
		<strong>Statistics and probability in biological data</strong> 
		</td>
		<td> <a class="email" href="https://sarwanpasha.github.io/Courses/ML_For_G/MF_2.pdf" target="_blank" style="color:blue">Slides</a>  </td>
                <td> <a> - </a> </td>
           </tr>	    

	    <tr>
                <td><a style="color:green">Topic 2.3: Mathematical Foundations</a>
		<br>
		<strong>Information theory basics (entropy, mutual information)</strong> 
		</td>
		<td> <a> - </a> </td>
                <td> <a> - </a> </td>
           </tr>	    

	    <tr>
                <td><a style="color:green">Topic 2.4: Mathematical Foundations</a>
		<br>
		<strong>Introduction to Python/R for genomics</strong> 
		</td>
		<td> <a> - </a> </td>
                <td> <a> - </a> </td>
           </tr>

	   <tr>
                <td><a style="color:violet">Topic 3.1: Data Preprocessing and Feature Engineering</a>
		<br>
		<strong>Genomic Data Types and Formats (FASTA, FASTQ, SAM/BAM, VCF, GFF/GTF formats)</strong>
		</td>
                <td> <a> - </a> </td>
                <td> <a> - </a> </td>
           </tr>

	   <tr>
                <td><a style="color:violet">Topic 3.2: Data Preprocessing and Feature Engineering</a>
		<br>
		<strong>Genomic Data Types and Formats (Quality control and preprocessing pipelines)</strong>
		</td>
                <td> <a> - </a> </td>
                <td> <a> - </a> </td>
           </tr>

	   <tr>
                <td><a style="color:violet">Topic 3.3: Data Preprocessing and Feature Engineering</a>
		<br>
		<strong>Genomic Data Types and Formats (Sequence alignment and variant calling basics)</strong>
		</td>
                <td> <a> - </a> </td>
                <td> <a> - </a> </td>
           </tr>

	   <tr>
                <td><a style="color:violet">Topic 3.4: Data Preprocessing and Feature Engineering</a>
		<br>
		<strong>Genomic Data Types and Formats (Data normalization techniques)</strong>
		</td>
                <td> <a> - </a> </td>
                <td> <a> - </a> </td>
           </tr>

	   <tr>
                <td><a style="color:violet">Topic 3.5: Data Preprocessing and Feature Engineering</a>
		<br>
		<strong>Feature Representation (k-mer representations and n-gram models)</strong>
		</td>
                <td> <a> - </a> </td>
                <td> <a> - </a> </td>
           </tr>

	   <tr>
                <td><a style="color:violet">Topic 3.6: Data Preprocessing and Feature Engineering</a>
		<br>
		<strong>Feature Representation (One-hot encoding for biological sequences)</strong>
		</td>
                <td> <a> - </a> </td>
                <td> <a> - </a> </td>
           </tr>

	   <tr>
                <td><a style="color:violet">Topic 3.7: Data Preprocessing and Feature Engineering</a>
		<br>
		<strong>Feature Representation (Position weight matrices (PWMs))</strong>
		</td>
                <td> <a> - </a> </td>
                <td> <a> - </a> </td>
           </tr>

	   <tr>
                <td><a style="color:violet">Topic 3.8: Data Preprocessing and Feature Engineering</a>
		<br>
		<strong>Feature Representation (Physicochemical properties as features)</strong>
		</td>
                <td> <a> - </a> </td>
                <td> <a> - </a> </td>
           </tr>

	   <tr>
                <td><a style="color:green">Topic 4.1: Classical Machine Learning in Genomics</a>
		<br>
		<strong>Supervised Learning Applications (Classification problems: gene function prediction, disease classification)</strong> 
		</td>
		<td> <a> - </a> </td>
                <td> <a> - </a> </td>
           </tr>

	   <tr>
                <td><a style="color:green">Topic 4.2: Classical Machine Learning in Genomics</a>
		<br>
		<strong>Supervised Learning Applications (Regression: gene expression prediction, quantitative trait analysis)</strong> 
		</td>
		<td> <a> - </a> </td>
                <td> <a> - </a> </td>
           </tr>

	   <tr>
                <td><a style="color:green">Topic 4.3: Classical Machine Learning in Genomics</a>
		<br>
		<strong>Supervised Learning Applications (Support vector machines for sequence classification)</strong> 
		</td>
		<td> <a> - </a> </td>
                <td> <a> - </a> </td>
           </tr>

	   <tr>
                <td><a style="color:green">Topic 4.4: Classical Machine Learning in Genomics</a>
		<br>
		<strong>Supervised Learning Applications (Random forests for genomic feature selection)</strong> 
		</td>
		<td> <a> - </a> </td>
                <td> <a> - </a> </td>
           </tr>

	   <tr>
                <td><a style="color:green">Topic 4.5: Classical Machine Learning in Genomics</a>
		<br>
		<strong>Unsupervised Learning (Principal component analysis (PCA) for population structure)</strong> 
		</td>
		<td> <a> - </a> </td>
                <td> <a> - </a> </td>
           </tr>

	   <tr>
                <td><a style="color:green">Topic 4.6: Classical Machine Learning in Genomics</a>
		<br>
		<strong>Unsupervised Learning (Clustering methods for gene expression analysis)</strong> 
		</td>
		<td> <a> - </a> </td>
                <td> <a> - </a> </td>
           </tr>

	   <tr>
                <td><a style="color:green">Topic 4.7: Classical Machine Learning in Genomics</a>
		<br>
		<strong>Unsupervised Learning (Hidden Markov Models (HMMs) for sequence analysis)</strong> 
		</td>
		<td> <a> - </a> </td>
                <td> <a> - </a> </td>
           </tr>

	   <tr>
                <td><a style="color:green">Topic 4.8: Classical Machine Learning in Genomics</a>
		<br>
		<strong>Unsupervised Learning (Dimensionality reduction techniques)</strong> 
		</td>
		<td> <a> - </a> </td>
                <td> <a> - </a> </td>
           </tr>

	   <tr>
                <td><a style="color:green">Topic 4.9: Classical Machine Learning in Genomics</a>
		<br>
		<strong>Ensemble Methods and Model Selection (Bagging and boosting in genomics)</strong> 
		</td>
		<td> <a> - </a> </td>
                <td> <a> - </a> </td>
           </tr>

	   <tr>
                <td><a style="color:green">Topic 4.10: Classical Machine Learning in Genomics</a>
		<br>
		<strong>Ensemble Methods and Model Selection (Cross-validation strategies for genomic data)</strong> 
		</td>
		<td> <a> - </a> </td>
                <td> <a> - </a> </td>
           </tr>

	   <tr>
                <td><a style="color:green">Topic 4.11: Classical Machine Learning in Genomics</a>
		<br>
		<strong>Ensemble Methods and Model Selection (Dealing with class imbalance)</strong> 
		</td>
		<td> <a> - </a> </td>
                <td> <a> - </a> </td>
           </tr>

	   <tr>
                <td><a style="color:green">Topic 4.12: Classical Machine Learning in Genomics</a>
		<br>
		<strong>Ensemble Methods and Model Selection (Feature selection and regularization)</strong> 
		</td>
		<td> <a> - </a> </td>
                <td> <a> - </a> </td>
           </tr>

	   <tr>
                <td><a style="color:violet">Topic 5.1: Deep Learning Fundamentals</a>
		<br>
		<strong>Neural Networks for Genomics (Perceptrons and multilayer networks)</strong>
		</td>
                <td> <a> - </a> </td>
                <td> <a> - </a> </td>
           </tr>

	   <tr>
                <td><a style="color:violet">Topic 5.2: Deep Learning Fundamentals</a>
		<br>
		<strong>Neural Networks for Genomics (Backpropagation algorithm)</strong>
		</td>
                <td> <a> - </a> </td>
                <td> <a> - </a> </td>
           </tr>

	   <tr>
                <td><a style="color:violet">Topic 5.3: Deep Learning Fundamentals</a>
		<br>
		<strong>Neural Networks for Genomics (Activation functions and their biological interpretations)</strong>
		</td>
                <td> <a> - </a> </td>
                <td> <a> - </a> </td>
           </tr>

	   <tr>
                <td><a style="color:violet">Topic 5.4: Deep Learning Fundamentals</a>
		<br>
		<strong>Neural Networks for Genomics (Overfitting and regularization techniques)</strong>
		</td>
                <td> <a> - </a> </td>
                <td> <a> - </a> </td>
           </tr>

	   <tr>
                <td><a style="color:violet">Topic 5.5: Deep Learning Fundamentals</a>
		<br>
		<strong>Convolutional Neural Networks (CNN architectures for sequence data)</strong>
		</td>
                <td> <a> - </a> </td>
                <td> <a> - </a> </td>
           </tr>

	   <tr>
                <td><a style="color:violet">Topic 5.6: Deep Learning Fundamentals</a>
		<br>
		<strong>Convolutional Neural Networks (Motif discovery using CNNs)</strong>
		</td>
                <td> <a> - </a> </td>
                <td> <a> - </a> </td>
           </tr>

	   <tr>
                <td><a style="color:violet">Topic 5.7: Deep Learning Fundamentals</a>
		<br>
		<strong>Convolutional Neural Networks (DeepBind and similar tools)</strong>
		</td>
                <td> <a> - </a> </td>
                <td> <a> - </a> </td>
           </tr>

	   <tr>
                <td><a style="color:violet">Topic 5.8: Deep Learning Fundamentals</a>
		<br>
		<strong>Convolutional Neural Networks (Transcription factor binding site prediction)</strong>
		</td>
                <td> <a> - </a> </td>
                <td> <a> - </a> </td>
           </tr>

	   <tr>
                <td><a style="color:violet">Topic 5.9: Deep Learning Fundamentals</a>
		<br>
		<strong>Recurrent Neural Networks (LSTM and GRU architectures)</strong>
		</td>
                <td> <a> - </a> </td>
                <td> <a> - </a> </td>
           </tr>

	   <tr>
                <td><a style="color:violet">Topic 5.10: Deep Learning Fundamentals</a>
		<br>
		<strong>Recurrent Neural Networks (Sequence-to-sequence models)</strong>
		</td>
                <td> <a> - </a> </td>
                <td> <a> - </a> </td>
           </tr>

	   <tr>
                <td><a style="color:violet">Topic 5.11: Deep Learning Fundamentals</a>
		<br>
		<strong>Recurrent Neural Networks (Protein secondary structure prediction)</strong>
		</td>
                <td> <a> - </a> </td>
                <td> <a> - </a> </td>
           </tr>

	   <tr>
                <td><a style="color:violet">Topic 5.12: Deep Learning Fundamentals</a>
		<br>
		<strong>Recurrent Neural Networks (RNA structure prediction)</strong>
		</td>
                <td> <a> - </a> </td>
                <td> <a> - </a> </td>
           </tr>

	   <tr>
                <td><a style="color:green">Topic 6.1: Advanced Deep Learning Applications</a>
		<br>
		<strong>Attention Mechanisms and Transformers (Self-attention in biological sequences)</strong> 
		</td>
		<td> <a> - </a> </td>
                <td> <a> - </a> </td>
           </tr>

	   <tr>
                <td><a style="color:green">Topic 6.2: Advanced Deep Learning Applications</a>
		<br>
		<strong>Attention Mechanisms and Transformers (BERT-like models for genomics (DNABERT, ProtBERT))</strong> 
		</td>
		<td> <a> - </a> </td>
                <td> <a> - </a> </td>
           </tr>

	   <tr>
                <td><a style="color:green">Topic 6.3: Advanced Deep Learning Applications</a>
		<br>
		<strong>Attention Mechanisms and Transformers (Positional encoding for genomic sequences)</strong> 
		</td>
		<td> <a> - </a> </td>
                <td> <a> - </a> </td>
           </tr>

	   <tr>
                <td><a style="color:green">Topic 6.4: Advanced Deep Learning Applications</a>
		<br>
		<strong>Attention Mechanisms and Transformers (Long-range dependency modeling)</strong> 
		</td>
		<td> <a> - </a> </td>
                <td> <a> - </a> </td>
           </tr>

	   <tr>
                <td><a style="color:green">Topic 6.5: Advanced Deep Learning Applications</a>
		<br>
		<strong>Generative Models (Variational autoencoders (VAEs) for genomic data)</strong> 
		</td>
		<td> <a> - </a> </td>
                <td> <a> - </a> </td>
           </tr>

	   <tr>
                <td><a style="color:green">Topic 6.6: Advanced Deep Learning Applications</a>
		<br>
		<strong>Generative Models (Generative adversarial networks (GANs))</strong> 
		</td>
		<td> <a> - </a> </td>
                <td> <a> - </a> </td>
           </tr>

	   <tr>
                <td><a style="color:green">Topic 6.7: Advanced Deep Learning Applications</a>
		<br>
		<strong>Generative Models (Sequence generation and design)</strong> 
		</td>
		<td> <a> - </a> </td>
                <td> <a> - </a> </td>
           </tr>

	   <tr>
                <td><a style="color:green">Topic 6.8: Advanced Deep Learning Applications</a>
		<br>
		<strong>Generative Models (Data augmentation techniques)</strong> 
		</td>
		<td> <a> - </a> </td>
                <td> <a> - </a> </td>
           </tr>

	   <tr>
                <td><a style="color:violet">Topic 7.1: Epigenetics and Regulatory Genomics</a>
		<br>
		<strong>Epigenetic Data Analysis (Histone modification patterns)</strong>
		</td>
                <td> <a> - </a> </td>
                <td> <a> - </a> </td>
           </tr>

	   <tr>
                <td><a style="color:violet">Topic 7.2: Epigenetics and Regulatory Genomics</a>
		<br>
		<strong>Epigenetic Data Analysis (DNA methylation analysis)</strong>
		</td>
                <td> <a> - </a> </td>
                <td> <a> - </a> </td>
           </tr>

	   <tr>
                <td><a style="color:violet">Topic 7.3: Epigenetics and Regulatory Genomics</a>
		<br>
		<strong>Epigenetic Data Analysis (HChromatin accessibility (ATAC-seq, FAIRE-seq))</strong>
		</td>
                <td> <a> - </a> </td>
                <td> <a> - </a> </td>
           </tr>

	   <tr>
                <td><a style="color:violet">Topic 7.4: Epigenetics and Regulatory Genomics</a>
		<br>
		<strong>Epigenetic Data Analysis (3D genome organization (Hi-C data))</strong>
		</td>
                <td> <a> - </a> </td>
                <td> <a> - </a> </td>
           </tr>

	   <tr>
                <td><a style="color:violet">Topic 7.5: Epigenetics and Regulatory Genomics</a>
		<br>
		<strong>Machine Learning for Regulatory Elements (Enhancer-promoter interaction prediction)</strong>
		</td>
                <td> <a> - </a> </td>
                <td> <a> - </a> </td>
           </tr>

	   <tr>
                <td><a style="color:violet">Topic 7.6: Epigenetics and Regulatory Genomics</a>
		<br>
		<strong>Machine Learning for Regulatory Elements (Chromatin state segmentation)</strong>
		</td>
                <td> <a> - </a> </td>
                <td> <a> - </a> </td>
           </tr>

	   <tr>
                <td><a style="color:violet">Topic 7.7: Epigenetics and Regulatory Genomics</a>
		<br>
		<strong>Machine Learning for Regulatory Elements (Epigenome-wide association studies (EWAS))</strong>
		</td>
                <td> <a> - </a> </td>
                <td> <a> - </a> </td>
           </tr>

	   <tr>
                <td><a style="color:violet">Topic 7.8: Epigenetics and Regulatory Genomics</a>
		<br>
		<strong>Machine Learning for Regulatory Elements (Multi-omics integration approaches)</strong>
		</td>
                <td> <a> - </a> </td>
                <td> <a> - </a> </td>
           </tr>

	   <tr>
                <td><a style="color:green">Topic 8.1: Single-Cell Genomics</a>
		<br>
		<strong>Single-Cell RNA Sequencing (scRNA-seq data characteristics and challenges)</strong> 
		</td>
		<td> <a> - </a> </td>
                <td> <a> - </a> </td>
           </tr>

	   <tr>
                <td><a style="color:green">Topic 8.2: Single-Cell Genomics</a>
		<br>
		<strong>Single-Cell RNA Sequencing (Dimensionality reduction (t-SNE, UMAP))</strong> 
		</td>
		<td> <a> - </a> </td>
                <td> <a> - </a> </td>
           </tr>

	   <tr>
                <td><a style="color:green">Topic 8.3: Single-Cell Genomics</a>
		<br>
		<strong>Single-Cell RNA Sequencing (Cell type identification and clustering)</strong> 
		</td>
		<td> <a> - </a> </td>
                <td> <a> - </a> </td>
           </tr>

	   <tr>
                <td><a style="color:green">Topic 8.4: Single-Cell Genomics</a>
		<br>
		<strong>Single-Cell RNA Sequencing (Pseudotime analysis and trajectory inference)</strong> 
		</td>
		<td> <a> - </a> </td>
                <td> <a> - </a> </td>
           </tr>

	   <tr>
                <td><a style="color:green">Topic 8.5: Single-Cell Genomics</a>
		<br>
		<strong>Deep Learning for Single-Cell Data (Autoencoders for dimensionality reduction)</strong> 
		</td>
		<td> <a> - </a> </td>
                <td> <a> - </a> </td>
           </tr>

	   <tr>
                <td><a style="color:green">Topic 8.6: Single-Cell Genomics</a>
		<br>
		<strong>Deep Learning for Single-Cell Data (Deep generative models (scVI, scGAN))</strong> 
		</td>
		<td> <a> - </a> </td>
                <td> <a> - </a> </td>
           </tr>

	   <tr>
                <td><a style="color:green">Topic 8.7: Single-Cell Genomics</a>
		<br>
		<strong>Deep Learning for Single-Cell Data (Batch effect correction)</strong> 
		</td>
		<td> <a> - </a> </td>
                <td> <a> - </a> </td>
           </tr>

	   <tr>
                <td><a style="color:green">Topic 8.8: Single-Cell Genomics</a>
		<br>
		<strong>Deep Learning for Single-Cell Data (Cell-cell communication inference)</strong> 
		</td>
		<td> <a> - </a> </td>
                <td> <a> - </a> </td>
           </tr>

	   <tr>
                <td><a style="color:violet">Topic 9.1: Integration and Applications</a>
		<br>
		<strong>Multi-Omics Integration (Data fusion strategies)</strong>
		</td>
                <td> <a> - </a> </td>
                <td> <a> - </a> </td>
           </tr>

	   <tr>
                <td><a style="color:violet">Topic 9.2: Integration and Applications</a>
		<br>
		<strong>Multi-Omics Integration (Network-based approaches)</strong>
		</td>
                <td> <a> - </a> </td>
                <td> <a> - </a> </td>
           </tr>

	   <tr>
                <td><a style="color:violet">Topic 9.3: Integration and Applications</a>
		<br>
		<strong>Multi-Omics Integration (Pathway analysis and functional annotation)</strong>
		</td>
                <td> <a> - </a> </td>
                <td> <a> - </a> </td>
           </tr>

	   <tr>
                <td><a style="color:violet">Topic 9.4: Integration and Applications</a>
		<br>
		<strong>Multi-Omics Integration (Precision medicine applications)</strong>
		</td>
                <td> <a> - </a> </td>
                <td> <a> - </a> </td>
           </tr>

	   <tr>
                <td><a style="color:violet">Topic 9.5: Integration and Applications</a>
		<br>
		<strong>Current Challenges and Future Directions (Interpretability in genomic deep learning)</strong>
		</td>
                <td> <a> - </a> </td>
                <td> <a> - </a> </td>
           </tr>

	   <tr>
                <td><a style="color:violet">Topic 9.6: Integration and Applications</a>
		<br>
		<strong>Current Challenges and Future Directions (Fairness and bias in genomic algorithms)</strong>
		</td>
                <td> <a> - </a> </td>
                <td> <a> - </a> </td>
           </tr>

	   <tr>
                <td><a style="color:violet">Topic 9.7: Integration and Applications</a>
		<br>
		<strong>Current Challenges and Future Directions (Privacy-preserving genomic analysis)</strong>
		</td>
                <td> <a> - </a> </td>
                <td> <a> - </a> </td>
           </tr>

	   <tr>
                <td><a style="color:violet">Topic 9.8: Integration and Applications</a>
		<br>
		<strong>Current Challenges and Future Directions (Large language models for genomics)</strong>
		</td>
                <td> <a> - </a> </td>
                <td> <a> - </a> </td>
           </tr>

        </tbody>
    </table>
	
	
</body></html>
